config/locales/en.yml

```diff
@@ -346,7 +346,7 @@ en:
     one: "<strong>%{period}</strong> your users opened 1 new ticket"
     other: "<strong>%{period}</strong> your users opened <strong>%{count} new tickets</strong>"
   insights_blurb_two: "of which your team replied to <strong>%{number_replied} (%{percentage_replied})</strong>"
-  insights_blurb_three: "with a median response time of <strong>%{response_time}</strong>"
+  insights_blurb_three_html_html: "with a median response time of <strong>%{response_time}</strong>"
```

Comment:
Hmmm this looks like your html got away from you!

Reply:
Yeah, I ran a script to replace the keys, must have editted the `en.yml` file manually and then again with the script.

Should be good now!